github-sync/github-data/users.yaml

```diff
@@ -77,6 +79,10 @@ users:
         role: maintainer
       - name: tuf-root-signing-codeowners
         role: maintainer
+      - name: sigstore-oncall
```

Comment:
When I initially added TSC members to `sigstore-oncall` it overwrote the `admin` permissions we already have on the various repos, so I ended up removing it. could you remove this until we figure out why?

Reply:
Whoops, good catch, didn't mean for this to be there! I think this showed up when I did a rebase